applications/plugins/snake_game/helpers/snake_file_handler.c

```diff
@@ -0,0 +1,181 @@
+#include "snake_file_handler.h"
+
+#include <furi.h>
+#include <flipper_format/flipper_format.h>
+
+static void snake_game_close_file(FlipperFormat* file) {
+    if(file == NULL) {
+        furi_record_close(RECORD_STORAGE);
+        return;
+    }
+    flipper_format_file_close(file);
+    flipper_format_free(file);
+    furi_record_close(RECORD_STORAGE);
+}
+
+static FlipperFormat* snake_game_open_file() {
+    Storage* storage = furi_record_open(RECORD_STORAGE);
```

Comment:
Opening and closing storage record doesn't looks good here. The best option is to do it in allocator/deallocator of snake app itself and then pass it here.